mysql-test/suite/innodb_fts/t/innodb_fts_misc_1.test

```diff
@@ -960,3 +960,16 @@ CREATE FULLTEXT INDEX i ON t1 (f2);
 SELECT * FROM t1 WHERE match(f2) against("bbb");
 # Cleanup
 DROP TABLE t1;
+
+--echo #
+--echo # MDEV-35937 sym_node->table != NULL in pars_retrieve_table_def
+--echo #
+CREATE TABLE t1 (a TEXT ,FULLTEXT (a)) ENGINE=INNODB;
+ALTER TABLE t1 DISCARD TABLESPACE;
+SET @save_ft = @@GLOBAL.innodb_ft_aux_table;
+SET GLOBAL innodb_ft_aux_table='test/t1';
+SELECT * FROM information_schema.innodb_ft_config;
+SELECT * FROM information_schema.innodb_ft_index_cache;
+SELECT * FROM information_schema.innodb_ft_index_table;
```

Comment:
I can’t reproduce any crash on 10.6 even if I move the `ALTER TABLE t1 DISCARD TABLESPACE` after the `SET GLOBAL`. Do you have any explanation why it might be so? Maybe a comment in the test would be useful.

Reply:
This is being fixed in 10.6:
```
commit 0aa2bc7a94d96397e2e061dacb8ed04a32c13fa0
Author: Marko Mäkelä <marko.makela@mariadb.com>
Date:   Thu Jul 29 16:41:58 2021 +0300

    MDEV-19445/MDEV-16678 fixup: Acquire proper MDL on innodb_ft_aux_table
    
    In commit 2647fd101db922b1e6c7363124adfc03e81ab8a0 (MDEV-19445)
    we fixed a race condition around the INFORMATION_SCHEMA tables
    that access the table identified by the global variable
    innodb_ft_aux_table. Thanks to MDEV-16678 we could fix it
    even better by using MDL instead of the InnoDB dict_sys latches.

```

dict_acquire_mdl_shared() does check whether the table is accessed or not.